pkg/policy/mapstate.go

```diff
@@ -93,6 +96,9 @@ type Key struct {
 	// DestPort is the port at L4 to / from which traffic is allowed, in
 	// host-byte order.
 	DestPort uint16
+	// PortMask is the mask that should be applied to the DestPort to
+	// define a range of ports that the policy-rule should be applied to.
+	PortMask uint16
```

Comment:
Please give a couple examples in here for the outcome of, for example, `DestPort: 100` and `PortMask: 0`; `DestPort: 100` and `PortMask: ffff`; `DestPort: 0` and `PortMask: ffff`;